scripts/devenv-builder/configure-vm.sh

```diff
@@ -47,6 +47,14 @@ if [ $RHEL_SUBSCRIPTION = true ] ; then
     fi
 fi
 
+# FIXME: Temporarily disable selinux until we update the version of
+# RHEL used in CI.
+if ! grep -q 'Red Hat Enterprise Linux release 9.2' /etc/redhat-release; then
+    echo "WARNING: disabling selinux on $(cat /etc/redhat-release)"
+    sudo setenforce Permissive
+    sudo sed --in-place -e 's/SELINUX=.*/SELINUX=permissive/g' /etc/selinux/config
```

Comment:
We need the `^SELINUX=` pattern, or the command will replace comments in the file.

Reply:
Fixed